wasmtime-runtime/src/lib.rs

```diff
@@ -32,6 +32,7 @@ extern crate failure_derive;
 mod export;
 mod imports;
 mod instance;
+mod jit_dump;
```

Comment:
Should this line have a `#[cfg(feature = jitdump)]`?

Reply:
For now I've removed the line in wasmtime-runtime/Cargo.toml. Please advise though if you'd like it back. I was discovering a cascading effects of errors that needed guarding.